get_databricks_cluster_eventlog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Default directory to save results to
+results_dir="$PWD/databricks_cluster_eventlogs"
+mkdir -p $results_dir
+
+function print_usage() {
+  cat <<-EOD
+	Usage: $0 -c <clusterId> [-r <results directory>]
+	Options:
+	  -c <cluster ID>  Databricks cluster ID
+	  -r <results directory>  directory in which to save cluster eventlog
+	  -h                      show this helpful message and exit
+	EOD
+}
+
+while getopts c:r:h: flag
+do
+    case "${flag}" in
+        c) clusterid="$OPTARG";;
+        r) results_dir="$OPTARG";;
+        h) print_usage && exit;;
+        *) >&2 print_usage && exit 1;;
+    esac
+done
+
+
+offset=0
+function eventlog_cli_call {
+    databricks clusters events --cluster-id $clusterid --output JSON --order ASC --offset $offset
+}
+
+
+# Make initial databricks call
+call=$(eventlog_cli_call)
+events=$(jq '.events' <<< $call)
+total_count=$(jq '.total_count' <<< $call)
```

Comment:
It might be safer to wrap each expansion in quotes in case the resulting value has a space in it, e.g.
```
events="$(jq .events <<<"$call")"
```

Reply:
Gotcha.  I wrapped everything except for the calls that returned a boolean or a integer.